modules/client-configuration/pages/clients-alibaba.adoc

```diff
@@ -0,0 +1,127 @@
+[[clients-alibaba]]
+= Registering {alibabaclo} Clients
+
+This section contains information about registering traditional and Salt clients running {alibabaclo} operating systems.
+
+ifeval::[{suma-content} == true]
+[IMPORTANT]
+====
+{alibabaclo} repository URLs are available from {scc}, but the packages and metadata are provided by Alibaba Cloud, not by {suse}.
+====
+endif::[]
+
+
+[IMPORTANT]
+====
+The traditional stack is available on {alibabaclo}{nbsp}2 but it is not supported.
+{alibabaclo}{nbsp}2 clients are only supported as Salt clients.
+====
+
+[WARNING]
+====
+Some {alibabaclo}{nbsp}2 instances will need two tries to register successfully.
+====
+
+== Add Software Channels
+
+Before you register {alibabaclo} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+ifeval::[{suma-content} == true]
```

Comment:
Needless to say, we'll need to revisit this if we ever decide to support Alibaba on SUSE Manager.

Reply:
Sure